ports/config/macathena-ssh-client-config/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           config_package 1.0
+
+github.setup        mit-athena ssh-client-config ee3aae0f0fec381288cc102aeccb8a612ea79166
+name                macathena-ssh-client-config
+version             1.7
+description         SSH client configuration for Macathena
+long_description    This package configures the SSH client to automatically forward \
+ Kerberos credentials.
+
+checksums           rmd160  fd3f0ac8cba58f9b38836680e3b42468e9f95177 \
+                    sha256  9c07af3db7c6f6a97fb370f9c8e78c303d8ad0993abb292c23c51140695526a5 \
+                    size    4761
+
+depends_run         port:macathena-kerberos-config \
+                    port:openssh
+
+post-activate {
+    system "(grep '^Include ssh_config.d/\*' ${prefix}/etc/ssh/ssh_config || echo 'Include ssh_config.d/*' >> ${prefix}/etc/ssh/ssh_config) > /dev/null 2>&1"
+}
+
+post-destroot {
+    file mkdir ${destroot}${prefix}/etc/ssh/ssh_config.d
+}
+
+config.install      ${worksrcpath}/ssh_known_hosts.debathena ${prefix}/etc/ssh/ssh_known_hosts
+config.files        ${prefix}/etc/ssh/ssh_config.d
```

Comment:
The destination should not contain $prefix; just `etc/ssh/ssh_known_hosts` and `etc/ssh/ssh_config.d`. (And likewise, `opt/local/` should not be part of the path in files.

Reply:
If I do etc/ssh/ssh_config.d it looks in filesetc: error copying "/opt/local/macathena/ports/config/macathena-ssh-client-config/filesetc/ssh/ssh_config.d". If I do /etc/ssh/ssh_config.d it puts it in /etc instead of /opt/local/etc. etc/ssh/ssh_known_hosts works though

Reply:
I think config_package-1.0.tcl should be ${filespath}/${file} in line 72